docs/src/manual/constraints.md

```diff
@@ -820,6 +819,38 @@ julia> is_valid(model, con)
 false
 ```
 
+Deleting a constraint does not unregister the symbolic reference from the model.
+Therefore, creating a new constraint of the same name will throw an error:
+```jldoctest constraints_delete
+julia> @constraint(model, con, 2x <= 1)
+ERROR: An object of name con is already attached to this model. If this
+    is intended, consider using the anonymous construction syntax, e.g.,
+    `x = @variable(model, [1:N], ...)` where the name of the object does
+    not appear inside the macro.
+
+    Alternatively, use `unregister(model, :con)` to first unregister
+    the existing name from the model. Note that this will not delete the
+    object; it will just remove the reference at `model[:con]`.
+[...]
+```
+
+After calling [`delete`](@ref), call [`unregister`](@ref) to remove the symbolic
```

Comment:
"After" -> "In addition to". It does not need to be done in a particular order

Reply:
It doesn't need the order, but I'd prefer it like this. `delete` is the main thing they're trying to do. `unregister` is an optional extra in some cases.